modules/standard/IO.chpl

```diff
@@ -2906,7 +2906,7 @@ pragma "no doc"
 proc channel.advance(amount:int(64)) {
   on this.home {
     this.lock();
-    var err = qio_channel_advance(false, _channel_internal);
+    var err = qio_channel_advance(false, _channel_internal, amount);
```

Comment:
I'm curious why this didn't throw an error before. Does our extern setup not catch a not enough arguments case, or is there another qio_channel_advance signature I'm not seeing?